config/services/common/services_prod.yml

```diff
@@ -0,0 +1,97 @@
+parameters:
```

Comment:
Should be moved in `src/PrestaShopBundle/Resources/config/services*`, wdyt @mickaelandrieu ?